### ISSUE TITLE
String array values lowercased during "in" query

### ISSUE THREAD

**Issue by Madd0g** [balderdashy/sails-postgresql#108]
I saw references to case sensitivity in the code, but couldn't figure out the exact conditions for this lowercasing issue.

I have a pg db, with a text column, I was trying to run an in query on it, but was getting no results.

     colls.myTable.find({ name: [ 'Some name' ] })

I was digging really hard to find a problem with the query, but then after a while noticed that it became `'some name'` before it left criteriaProcessor.js.

**Comment by sailsbot** [comment]
Thanks for posting, @Madd0g.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
